ui/src/PaperSummary.tsx

```diff
@@ -140,13 +151,25 @@ export class PaperSummary extends React.PureComponent<
                   >
                     Cite
                   </Button>
-                  <Button
-                    startIcon={<SaveIcon />}
-                    className="paper-summary__action"
-                    onClick={() => warnOfUnimplementedActionAndTrack("save")}
-                  >
-                    Save
-                  </Button>
+                  {inLibrary ?
+                    <Button
+                      startIcon={<SaveIcon />}
+                      className="paper-summary__action"
+                      onClick={() => goToLibrary()}>
+                      In Your Library
+                    </Button> :
+                    <Button
```

Comment:
These 2 buttons are pretty similar besides the onclick and label, consider a function that takes in these 2 args and renders the component